core-plugins/widgets/Joiner-batchjoiner.json

```diff
@@ -0,0 +1,46 @@
+{
+  "metadata": {
+    "spec-version": "1.3"
+  },
+  "inputs": {
+    "multipleInputs": true
```

Comment:
Does this property mean we only allow more than 2 inputs for joiner plugin and not any other plugin?